pelops/features/hog.py

```diff
@@ -20,14 +21,38 @@ def produce_features(self, chip):
         img = self.get_image(chip)
         img = img.resize(self.image_size, Image.BICUBIC)
         img_x, img_y = img.size
+
+        # Calculate histogram of each channel
```

Comment:
i was only attempting to get all the relevant info up top so ppl don't have to dig.